builtin/logical/pki/fields.go

```diff
@@ -250,12 +250,22 @@ If you want more than one, specify alternative names in the alt_names
 map using OID 2.5.4.5. This has no impact on the final certificate's
 Serial Number field.`,
 	}
+
 	fields["not_after"] = &framework.FieldSchema{
 		Type: framework.TypeString,
 		Description: `Set the not after field of the certificate with specified date value.
 The value format should be given in UTC format YYYY-MM-ddTHH:MM:SSZ`,
 	}
 
+	fields["not_before_duration"] = &framework.FieldSchema{
+		Type:        framework.TypeDurationSecond,
+		Default:     30,
+		Description: `The duration before now the cert needs to be created / signed.`,
```

Comment:
I don't understand this description.

Reply:
That's actually consistent with the existing description. I think perhaps we should update both... Maybe 

> The duration before now which the certificate needs to be backdated by.

is clearer?

Reply:
Yeah, that's much clearer, thank you